### DISCUSSION TITLE
I would like to propose a Side Menu component

### DISCUSSION THREAD

**Discussion by Altynay18** [segmentio/evergreen#1420] [Ideas] [answered]
Dear contributors,
I have noticed that in the components list there is no `Side Menu` component. I have seen quite similar but different in usage component named Side Sheet. My idea is to create Side Menu component: it is basically the combination of existing Side Sheet and Menu items components. What do you think about it?

**Comment by brandongregoryscott** [comment, marked as answer]
Hi @Altynay18, thanks for the idea. We haven't had any use cases for a component like this yet, but it should be easy enough to compose using the two components.
